File: Modul 4 Sem 1/C/PRAK405-2410817110005-MuhammadRizqanKarim.c
#include <stdio.h>

int main() {
    int n, kelipatan, total = 0;

    printf("Masukkan banyaknya baris: ");
    scanf("%d", &n);
    printf("Masukkan kelipatan: ");
    scanf("%d", &kelipatan);

    for (int i = 1; i <= n; i++) {
        int hasil = 0;
        printf("(");
        for (int j = 1; j <= i; j++) {
            printf("%d * %d", j, kelipatan);
            hasil += j * kelipatan;
            if (j < i) {
                printf(") + (");
            }
        }
        printf(") = %d\n", hasil);
        total += hasil;
    }
    printf("%d\n", total);

    return 0;
}

File: Modul 4 Sem 1/PHYTHON/PRAK405-2410817110005-MuhammadRizqanKarim.py
n = int(input("Masukkan banyaknya baris: "))
kelipatan = int(input("Masukkan kelipatan: "))

total = 0
for i in range(1, n + 1):
    baris = " + ".join([f"({j} * {kelipatan})" for j in range(1, i + 1)])
    hasil = sum(j * kelipatan for j in range(1, i + 1))
    total += hasil
    print(f"{baris} = {hasil}")
print(total)
